### PR TITLE
Tooltips: additions & rephrasings

### PR DESCRIPTION
**Pull request by ronso0**
additions, most of them for LateNight:
* big_spinny_coverart
* show_library_coverart
* show_waveforms
* show_beatgrid_controls
* show_vumeters

rephrased:
* talkover_duck_mode
* talkover_duck_strength

### CODE REVIEW

**Comment by ronso0** [comment]
I'm not sure if the rephrasings violate string freeze. if so I can break them out and target at master.

File: src/skin/tooltips.cpp [hunk 1]
```diff
@@ -256,12 +270,17 @@ void Tooltips::addStandardTooltips() {
 
     add("maximize_library")
             << tr("Maximize Library")
-            << tr("Maximize the track library to take up all the available screen space.");
+            << tr("Hide all skin sections except the decks to have more")
+            << tr("screen space for the track library.");
```
Comment (Holzhaus) [inline src/skin/tooltips.cpp:274]: Will these show up as seperate translations?

If so, you can also do this:
```suggestion
            << tr("Hide all skin sections except the decks to have more "
                  "screen space for the track library.");
```

Comment (ronso0) [reply src/skin/tooltips.cpp]: oh, I see they can easily be on one line.

File: src/skin/tooltips.cpp [hunk 2]
```diff
@@ -70,6 +71,10 @@ void Tooltips::addStandardTooltips() {
             << dropTracksHere
             << tr("If Vinyl control is enabled, displays time-coded vinyl signal quality (see Preferences -> Vinyl Control).");
 
+    add("big_spinny_coverart")
+            << tr("Big Spinny/Cover Art")
+            << tr("Show a big version of the Spinny or track cover art if enabled.");
```
Comment (Holzhaus) [inline src/skin/tooltips.cpp:76]: These new tooltips do now show up for me. I guess this is because you didn't add them to LateNight yet?

Comment (ronso0) [reply src/skin/tooltips.cpp:76]: exactly. this about the strings only, the implementation will happen elsewhere.

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you!

**Pull request merged**